.evergreen/config.yml

```diff
@@ -2258,10 +2258,10 @@ task_groups:
           shell: "bash"
           script: |
             ${PREPARE_SHELL}
-            echo '${testgcpkms_key_file}' > /tmp/testgcpkms_key_file.json
+            echo '${testgcpkms_key_file_2}' > /tmp/testgcpkms_key_file.json
             export GCPKMS_KEYFILE=/tmp/testgcpkms_key_file.json
             export GCPKMS_DRIVERS_TOOLS=$DRIVERS_TOOLS
-            export GCPKMS_SERVICEACCOUNT="${testgcpkms_service_account}"
+            export GCPKMS_SERVICEACCOUNT="${testgcpkms_service_account_2}"
```

Comment:
TODO: before merge, swap the project variables in https://spruce.mongodb.com/project/mongo-go-driver/settings/variables.